sklearn/linear_model/_base.py

```diff
@@ -472,6 +472,15 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
     copy_X : bool, default=True
         If True, X will be copied; else, it may be overwritten.
 
+    tol : float, default=1e-4
+        The precision of the solution (`coef_`) is determined by `tol` which
+        specifies a different convergence criterion for the `lsqr` solver.
+        `tol` is set as atol and btol of scipy.sparse.linalg.lsqr,
+        which control the norm of the residual vector in terms of the norms of
+        matrix and coefficients.
```

Comment:
I think we can let the user read the details about the meaning of `atol` and `btol` in the scipy doc directly.

But we need to be more explicit that this only impacts training with sparse data.

```suggestion
        specifies a different convergence criterion for the `lsqr` solver.
        `tol` is set as `atol` and `btol` of `scipy.sparse.linalg.lsqr` when
        fitting on sparse training data. This parameter has no effect when fitting
        on dense data.
```